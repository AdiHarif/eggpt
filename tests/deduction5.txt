Sheep are afraid of cats.
Jessica is a sheep.
Mice are afraid of sheep.
Cats are afraid of sheep.
Wolves are afraid of mice.
Emily is a sheep.
Gertrude is a sheep.
Winona is a mouse.
What is winona afraid of?
What is jessica afraid of?
What is gertrude afraid of?
What is gertrude afraid of?