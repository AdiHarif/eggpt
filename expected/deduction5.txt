What is winona afraid of?	sheep	8 3
What is jessica afraid of?	cat	2 1
What is gertrude afraid of?	cat	7 1
What is gertrude afraid of?	cat	7 1